presto-main/src/main/java/com/facebook/presto/server/QueryStateInfoResource.java

```diff
@@ -53,29 +57,54 @@ public QueryStateInfoResource(
     }
 
     @GET
-    public List<QueryStateInfo> getQueryDiagnosticsByUser(@QueryParam("user") String user)
+    public List<QueryStateInfo> getQueryStateInfos(@QueryParam("user") String user)
```

Comment:
I missed this before, but these endpoints should declare that they produce JSON `@Produces(MediaType.APPLICATION_JSON)`